addons/website/views/website_views.xml

```diff
@@ -273,6 +273,10 @@
                     <field name="page_ids" invisible="1" />
                     <field name="first_page_id" attrs="{'invisible': [('page_ids', '=', [])]}" />
                     <field name="visibility" attrs="{'invisible': [('type', '!=', 'qweb')]}" />
+                    <div class="o_td_label" attrs="{'invisible': [('visibility', '!=', 'restricted_group')]}"></div>
+                    <div class="oe_edit_only alert alert-info" role="alert" attrs="{'invisible': [('visibility', '!=', 'restricted_group')]}">
+                        Add groups in the "Access Rights" tab below.
+                    </div>
```

Comment:
@rdeodoo The right way to do this (tested in 15.0 to be sure) would be this:
```suggestion
                    <label for="visibility"/>
                    <div>
                        <field name="visibility" attrs="{'invisible': [('type', '!=', 'qweb')]}" />
                        <div class="oe_edit_only alert alert-info" role="alert" attrs="{'invisible': [('visibility', '!=', 'restricted_group')]}">
                            Add groups in the "Access Rights" tab below.
                        </div>
                    </div>
```

Hardcoding "o_td_label" is a big hack... are you sure it is worth doing in 15.0? Also I am not sure about the changes for form views every step of the way: for exemple your attrs stuff will have to be changed in 16.X (which one?). Would not it be easier to just target 17.0 with improvements? :grimacing:

Reply:
I adapted to Odoo 17 with the same rationale: only diff addition and before `visibility_password_display` to not be in between `visibility_password_display` and a potential custo which would add its own fields after the visibility ones.

If there is a better alternative and more "form view framework friendly", I did not find one.
I ended up using `o_wrap_label` because anything else would fail:
- `<label/>` tag is forbidden is there is no `for` attribute, and we don't want to set one (even if empty text) because in debug there is the `?` showing / floating
- other tag and/or class would lead to `width: 50%` forced on both element (label and alert). It's actually when debugging this `50%` that I found this code:
https://github.com/odoo/odoo/blob/99fa3ec267dd1371a5f8c7a66fae39be45b29be6/addons/web/static/src/views/form/form_compiler.js#L333-L342
https://github.com/odoo/odoo/blob/99fa3ec267dd1371a5f8c7a66fae39be45b29be6/addons/web/static/src/views/form/form_group/form_group.xml#L48
https://github.com/odoo/odoo/blob/99fa3ec267dd1371a5f8c7a66fae39be45b29be6/addons/web/static/src/views/form/form_group/form_group.js#L112

Surely there is a "standard" way to add a component in only the right part of the row.. But I can't find it. I'll ask the framework if this does sounds too hacky.

Reply:
> But maybe your remark was implying that it was the correct way if we were actually not considering the stable part 👍

Of course. My point was: rather than doing hacks in a very old stable with the risk that the hack could be different in every stable on the way to master -> why not targeting 17.0 only.

> I adapted to Odoo 17 with the same rationale: only diff addition

Yes of course, that's why I would expect: but just not lose time every step to master for this.

> Surely there is a "standard" way to add a component in only the right part of the row

I did not follow the changes that closely, probably related to their 16.0 refactoring with groups :shrug: Nothing in the documentation? Should I wait you contact them / double check before checking this PR again then? :)

Reply:
The framework-js [seems off today](https://discord.com/channels/678381219515465750/687338039717920792/1202206033595093042), I'll let you know

Reply:
AFAIC, this is good to go as-is. I didn't find any better way except going for the empty text label with a `for` attribute `<label for=visibility/>` but it would show a random floating icon `?` and that area would also be clickable to focus the select above, really seems worst than the way I wrote it currently?
Even if it doesn't sound that bad to the framework-js to use the label solution, see https://discord.com/channels/678381219515465750/687338039717920792/1202511254388744253